test/e2e/cassandra_test.go

```diff
@@ -85,17 +88,27 @@ func (suite *CassandraTestSuite) TestCassandraSparkDependencies() {
 	require.NoError(t, err, "SparkTest failed")
 }
 
-func getJaegerWithCassandra(s string) *v1.Jaeger {
+func waitForCassandra() {
+	err := WaitForStatefulset(t, fw.KubeClient, storageNamespace, "cassandra", retryInterval, timeout)
+	require.NoError(t, err, "Error waiting for cassandra")
+}
+
+func getJaegerWithCassandra(jaegerInstanceName, namespace string) *v1.Jaeger {
+	ingressEnabled := true
 	j := &v1.Jaeger{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       "Jaeger",
 			APIVersion: "jaegertracing.io/v1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "with-cassandra",
+			Name:      jaegerInstanceName,
 			Namespace: namespace,
 		},
 		Spec: v1.JaegerSpec{
+			Ingress: v1.JaegerIngressSpec{
+				Enabled:  &ingressEnabled,
+				Security: v1.IngressSecurityNoneExplicit,
```

Comment:
Why is this needed? For OpenShift? We should really try to get the OpenShift service-ca trusted by the client. If it's not feasible for this PR, please create an issue to track it.